libc/src/__support/fixed_point/fx_bits.h

```diff
@@ -19,6 +21,36 @@
 
 namespace LIBC_NAMESPACE::fixed_point {
 
+// Bit-wise operations are not available for fixed point types yet.
+template <typename T>
+LIBC_INLINE constexpr cpp::enable_if_t<cpp::is_fixed_point_v<T>, T>
+bit_and(T x, T y) {
+  using BitType = typename FXRep<T>::StorageType;
+  static_assert(sizeof(BitType) * CHAR_BIT == sizeof(T) * CHAR_BIT);
+  BitType x_bit = cpp::bit_cast<BitType>(x);
+  BitType y_bit = cpp::bit_cast<BitType>(y);
+  return cpp::bit_cast<T, BitType>(x_bit & y_bit);
+}
+
+template <typename T>
+LIBC_INLINE constexpr cpp::enable_if_t<cpp::is_fixed_point_v<T>, T>
+bit_or(T x, T y) {
+  using BitType = typename FXRep<T>::StorageType;
+  static_assert(sizeof(BitType) * CHAR_BIT == FXRep<T>::TOTAL_LEN);
```

Comment:
Why something different here ?

Reply:
Oops, those `static_assert` are for debugging that I forgot to remove.  The issue was that for some reason, `bit_cast<fixed point>(x_bits)` cannot infer `BitType`, and keep putting `int` there and failed.  That's why I put the `From` explicitly to resolve the issue.

Reply:
Removed debugging assertion.